src/init.cpp

```diff
@@ -1218,6 +1218,7 @@ bool AppInitMain(NodeContext& node)
         // Not categorizing as "Warning" because it's the default behavior
         LogPrintf("Config file: %s (not found, skipping)\n", config_file_path.string());
     }
+    gArgs.LogArgs();
```

Comment:
nit: this is commented in the unit test but not here; perhaps add:
```diff
+
+    // Log the config arguments to debug.log
     gArgs.LogArgs();
```

Reply:
It seems this comment does not add any value here, no?

Reply:
I think it may gain time for occasional contributors, and perhaps for those who haven't reviewed this PR, by confirming what the laconic (and somewhat generic) `gArgs.LogArgs();` is doing exactly without the need to git grep.